serverlessworkflow/modules/ROOT/pages/cloud/build-and-deploy-with-serverless-operator-on-kubernetes.adoc

```diff
@@ -0,0 +1,178 @@
+----
+
+[NOTE]
+====
+To speed up, you can increase cpus and memory options. For example, use `--cpus 12 --memory 16384`.
+====
+
+[TIP]
+====
+If it does work with the default driver, aka `docker`, you can try to start with the `podman` driver:
+
+
+[source,shell,subs="attributes+"]
+----
+minikube start [...] --driver podman
+----
+====
+
+[IMPORTANT]
+====
+There are some issues with the `crio` container runtime and Kaniko that the operator is using. Reference: link:{kaniko_issue_url}[ISSUE-2201]
+====
+
+.Setup {product_name} Serverless operator
+
+In order to have an up-and-running instance of the {product_name} Serverless Operator you can use the following command:
+
+[source,shell,subs="attributes+"]
+----
+kubectl create -f https://raw.githubusercontent.com/kiegroup/kogito-serverless-operator/main/operator.yaml
+----
+
+[NOTE]
+====
+Follow the deployment of the {product_name} Serverless Operator:
+
++
+[source,shell,subs="attributes+"]
+----
+kubectl get pod -n kogito-serverless-operator-system --watch
+----
+
+You can also follow the operator’s log:
+
++
+[source,shell,subs="attributes+"]
+----
+kubectl logs deployment/kogito-serverless-operator-controller-manager -n kogito-serverless-operator-system -f
+----
+====
+
+Once the operator is running, it will watch for new custom resources so that you can prepare your environment to be ready to build new {context} based on the definitions you will send to the operator.
+
+.Preparing for the build
+
+Follow these steps to create a container that you can deploy as a service on Kubernetes or KNative.
+
+* Create a namespace for the building phase
+
+Let's create a new namespace that will hold all the resources that we (or the operator) will create (pods, deployments, services, secretes, config map, and custom resources) in this guide.
+
+[source,bash,subs="attributes+"]
+----
+kubectl create namespace kogito-workflows
+----
+* Create a secret for the container registry authentication
+[source,bash,subs="attributes+"]
+----
+kubectl create secret docker-registry regcred --docker-server=<registry_url> --docker-username=<registry_username> --docker-password=<registry_password> --docker-email=<registry_email> -n kogito-workflows
+----
+or you directly import your local docker config into your kubernetes cluster:
+[source,bash,subs="attributes+"]
+----
+kubectl create secret generic regcred --from-file=.dockerconfigjson=${HOME}/.docker/config.json --type=kubernetes.io/dockerconfigjson -n kogito-workflows
+----
+* Create a platform containing the configuration (i.e. registry address, secret) for building your workflows
+
+The Platform CR is the resource used to control the behavior of the {product_name} {context} operator.
+
+It defines the behavior of all Custom Resources (Workflow and Build) in the given namespace.
+
+Since the {product_name} {context} operator is installed in global mode, you will need to specify a Platform CR in each namespace where you want the operator to be executed.
```

Comment:
```suggestion
Since the {product_name} Serverless Operator is installed in global mode, you will need to specify a {product_name} Serverless Platform CR in each namespace where you want the operator to be executed.
```